apps/backoffice-v2/src/common/components/molecules/BusinessReportSummary/BusinessReportSummary.tsx

```diff
@@ -36,6 +37,7 @@ export const BusinessReportSummary: FunctionComponent<{
         <CardHeader className={'pt-4 font-bold'}>
           <span className={'mb-1'}>Overall Risk Level</span>
           <div className="flex items-center space-x-2">
+            {!riskScore && <TextWithNAFallback className={'text-4xl font-bold'} />}
```

Comment:
Re-added the empty state that was before we removed risk score. Same for instances below.